foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigurationSpringInitializer.java

```diff
@@ -63,6 +66,22 @@ public void setEnvironment(Environment environment) {
     ConfigUtil.addExtraConfig(EXTRA_CONFIG_SOURCE_PREFIX + environmentName, extraConfig);
 
     ConfigUtil.installDynamicConfig();
+
+    setUpSpringPropertySource(environment);
+  }
+
+  private void setUpSpringPropertySource(Environment environment) {
+    ConfigCenterConfigurationSource configCenterConfigurationSource =
```

Comment:
only add cc configuration to environment cause ConfigurationProperties  can not read configuration from microservice.yaml

Reply:
fixed and test updated :  https://github.com/huaweicse/cse-java-chassis-samples/commit/b25a690f221b734b7b4f6e477d82b5b6cfc06916